### PR TITLE
Cc 21 display menu options

### PR DESCRIPTION
**Pull request by AshokkSS**
Completed ticket as described.
Menu options are clearly displayed.

### CODE REVIEW

File: .results.json [hunk 1]
```diff
@@ -0,0 +1 @@
....rvm/gems/ruby-3.0.0/gems/tty-prompt-0.23.1/lib/tty/prompt/list.rb:245:in `block in call'\u001b[0m\n\u001b[36m# /Users/ashokksaravanabavan/.rvm/gems/ruby-3.0.0/gems/tty-reader-0.9.0/lib/tty/reader.rb:118:in `subscribe'\u001b[0m\n\u001b[36m# /Users/ashokksaravanabavan/.rvm/gems/ruby-3.0.0/gems/tty-prompt-0.23.1/lib/tty/prompt/list.rb:244:in `call'\u001b[0m\n\u001b[36m# /Users/ashokksaravanabavan/.rvm/gems/ruby-3.0.0/gems/tty-prompt-0.23.1/lib/tty/prompt.rb:249:in `invoke_select'\u001b[0m\n\u001b[36m# /Users/ashokksaravanabavan/.rvm/gems/ruby-3.0.0/gems/tty-prompt-0.23.1/lib/tty/prompt.rb:286:in `select'\u001b[0m\n\u001b[36m# ./lib/run.rb:26:in `display_main_menu'\u001b[0m\n\u001b[36m# ./spec/run_spec.rb:10:in `block (2 levels) in <top (required)>'\u001b[0m\n\u001b[36m# ./lib/run.rb:9:in `capture_stdout'\u001b[0m\n\u001b[36m# ./spec/run_spec.rb:9:in `block in <top (required)>'\u001b[0m\n\u001b[36m# ./spec/run_spec.rb:7:in `<top (required)>'\u001b[0m\n","No examples found."],"examples":[],"summary":{"duration":0.0001,"example_count":0,"failure_count":0,"pending_count":0,"errors_outside_of_examples_count":1},"summary_line":"0 examples, 0 failures, 1 error occurred outside of examples"}
```
Comment (Dark-Raine) [inline .results.json]: Do we need to store this file in git?

File: lib/run.rb [hunk 2]
```diff
@@ -1,34 +1,101 @@
 require_relative '../config/environment'
 CLEAR = "\e[H\e[2J"
+PROMPT = TTY::Prompt.new
+OPTIONS =  %w(Price_Check Compare_Cryptos My_Portfolio Close)
+PING_URL = "https://api.coingecko.com/api/v3/ping"
```
Comment (Dark-Raine) [inline lib/run.rb]: Could this live in it's own class where it's domain is just to do with connecting with the poke api?

File: lib/run.rb [hunk 3]
```diff
@@ -1,34 +1,101 @@
 require_relative '../config/environment'
 CLEAR = "\e[H\e[2J"
+PROMPT = TTY::Prompt.new
+OPTIONS =  %w(Price_Check Compare_Cryptos My_Portfolio Close)
```
Comment (Dark-Raine) [inline lib/run.rb]: Could these options not be bart of the main menu class?

File: lib/run.rb [hunk 4]
```diff
@@ -1,34 +1,101 @@
...
     
+    def ping_api
+        uri = URI(PING_URL)
+        res = Net::HTTP.get_response(uri)
+        if res.is_a?(Net::HTTPSuccess)
+            connected_box_success
+        else
+            connected_box_failure
+        end
+    end
```
Comment (Dark-Raine) [inline lib/run.rb]: This could be apart of the poke api class mentioned above

File: lib/run.rb [hunk 5]
```diff
@@ -1,34 +1,101 @@
...
+        sleep 2
+        puts CLEAR
+    end
+
+    def connected_box_failure
+        box = TTY::Box.error("Unable to connect to API, Please close and reopen.")
+        print box
+        sleep 2
+        puts CLEAR
+    end
```
Comment (Dark-Raine) [inline lib/run.rb]: Same thing for these

File: lib/run.rb [hunk 6]
```diff
@@ -1,34 +1,101 @@
...
+end
+
+# def capture_stdout(&blk)
+#     old = $stdout
+#     $stdout = fake = StringIO.new
+#     blk.call
+#     fake.string
+#   ensure
+#     $stdout = old
+#   end
```
Comment (Dark-Raine) [inline lib/run.rb]: delete commented code

File: lib/run.rb [hunk 7]
```diff
@@ -1,34 +1,101 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (Dark-Raine) [inline lib/run.rb]: remove whitespace from files

File: spec/run_spec.rb [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+# end
+# context "#display_main_menu" do
+#   it 'should print "my message"' do
+#     printed = capture_stdout do
+#       display_main_menu # do your actual method call
+#     end
+
+#     printed.should eq("my message")
+#   end
+# end
```
Comment (Dark-Raine) [inline spec/run_spec.rb]: delete commented out code

**Review by Dark-Raine — CHANGES_REQUESTED** [review]

**Comment by Dark-Raine** [comment]
Another thing to add, you haven't commited your gemfile and gemfile.lock. These are important in ensuring you always use the correct versions of the gems in your application

**Comment by AshokkSS** [comment]
Have made those changes now, just a few things.
I have added *.json to the gitignore but can still see it in my repo.
I can see the gem file is in the git?

**Review by Dark-Raine — APPROVED** [review]

**Pull request merged**